templates/minimal/README.md

```diff
@@ -147,6 +153,11 @@ docker run --rm polkadot-sdk-minimal-template
 We can install `zombienet` as described [here](https://paritytech.github.io/zombienet/install.html#installation),
 and `zombienet-omni-node.toml` contains the network specification we want to start.
 
+To simplify the process of starting the minimal template with zombienet and Omni Node, we've added a pre-configured 
+development chain spec (dev_chain_spec.json) to the minimal template. The zombienet-omni-node.toml file of this template
+points to it, but you can update it to an updated path to the new chain spec generated on your machine, as shown below.
```

Comment:
```suggestion
To simplify the process of starting the minimal template with zombienet and Omni Node, we've added a pre-configured 
development chain spec (dev_chain_spec.json) to the minimal template. The zombienet-omni-node.toml file of this template
points to it, but you can update it to an updated path to the new chain spec generated on your machine.
```

My initial suggestion was to move this right into the section: `Update zombienet-omni-node.toml with a valid chain spec path`. It can be placed right at the beginning of the section, but then I would change the next paragraph: `Before starting...` by removing the first sentence.

Reply:
I made the changes to this